modules/misc/nix.nix

```diff
@@ -221,6 +247,13 @@ in {
       '';
     }];
 
+    home.sessionVariables = mkIf (!cfg.keepOldNixPath) { inherit NIX_PATH; };
+
+    home.sessionVariablesExtra =
+      mkIf (cfg.keepOldNixPath && cfg.nixPath != [ ]) ''
+        export NIX_PATH="${NIX_PATH}''${NIX_PATH:+:$NIX_PATH}"
```

Comment:
cfg.nixPath is not used to generate NIX_PATH ?

Reply:
On line 240:

```
config = mkIf cfg.enable (let NIX_PATH = concatStringsSep ":" cfg.nixPath;
```

Reply:
could you move it closer to where it's used please ? makes code easier to read

Reply:
could you move it closer to where it's used please ? makes code easier to read

Reply:
Fixed :+1: